source/content/drupal-9.md

```diff
@@ -19,6 +19,12 @@ Pantheon engineers are rolling out changes often.
 
 </Alert>
 
+## Before You Begin
+
+The steps in this doc require command line tools on your local machine. You'll also need to be logged in to your Pantheon account. If you don't have one, [create one first](https://pantheon.io/register?docs) and familiarize yourself with the [User Dashboard](/guides/quickstart/user-dashboard) before you create a new site.
```

Comment:
Is Terminus explicitly required for this?

Reply:
good catch - I think of it as a "just do it while you're in the cli"
I'll remove that while I fix whatever the CI just caught

Reply:
tagging @carolyn-shannon in on this - she picked up this into to make it a little less daunting